torch/jit/__init__.py

```diff
@@ -399,6 +402,33 @@ def script_method(fn):
     return ScriptMethodStub(createResolutionCallback(frames_up=2), get_jit_ast(fn), fn)
 
 
+class batch(object):
+    def __init__(self, batch_size=1, optimize=True, _frames_up=0):
+        self.batch_size = batch_size
+        self.optimize = optimize
+        self._frames_up = _frames_up
+
+    def __call__(self, fn):
+        mod = script(fn, self.optimize, self._frames_up)
+        mod.to_batch(self.batch_size)
+        @functools.wraps(fn)
```

Comment:
Does this do anything other than forward `__doc__`? I think we moved the other callsite for this to just explicitly forward that attribute